aiobungie/internal/factory.py

```diff
@@ -1882,13 +1882,13 @@ def _deserialize_post_activity_player(
             destiny_user=self.deserialize_destiny_membership(
                 payload["player"]["destinyUserInfo"]
             ),
-            character_class=payload["player"]["characterClass"],
-            character_level=int(payload["player"]["characterLevel"]),
-            race_hash=int(payload["player"]["raceHash"]),
-            gender_hash=int(payload["player"]["genderHash"]),
-            light_level=int(payload["player"]["lightLevel"]),
-            emblem_hash=int(payload["player"]["emblemHash"]),
-            class_hash=payload["player"]["classHash"],
+            character_class=payload["player"].get("characterClass"),
```

Comment:
This field should not be optional as the docs doesn't [say](https://bungie-net.github.io/multi/schema_Destiny-HistoricalStats-DestinyPlayer.html#schema_Destiny-HistoricalStats-DestinyPlayer) they're nullable, So how did it raise a KeyError?